packages/block-library/src/image/edit.js

```diff
@@ -351,13 +352,19 @@ class ImageEdit extends Component {
 						onChange={ this.onSetLinkDestination }
 					/>
 					{ linkDestination !== LINK_DESTINATION_NONE && (
-						<TextControl
-							label={ __( 'Link URL' ) }
-							value={ href || '' }
-							onChange={ this.onSetCustomHref }
-							placeholder={ ! isLinkURLInputDisabled ? 'https://' : undefined }
-							disabled={ isLinkURLInputDisabled }
-						/>
+						<Fragment>
+							<TextControl
+								label={ __( 'Link URL' ) }
+								value={ href || '' }
+								onChange={ this.onSetCustomHref }
+								placeholder={ ! isLinkURLInputDisabled ? 'https://' : undefined }
+								disabled={ isLinkURLInputDisabled }
+							/>
+							<ToggleControl
+								label={ __( 'Open in New Window' ) }
```

Comment:
Worth considering in core all occurrences of `(opens in a new window)` have been changed to `(opens in a new tab)`, see https://core.trac.wordpress.org/ticket/43803 which will be released in 5.1 (not 5.0). Even if that string is meant for visually hidden accessible text, for consistency I'd suggest to refer to "tab" and not "window".